docs/modules/datadog_aws_integration.md

```diff
@@ -79,4 +88,4 @@ EOF
 The following arguments are supported:
 
 * `host_tags` - (optional) A map of strings to be applied as tags to each metric ingested through this integration. The key-value pair will be converted to a single tag formatted like "{k}:{v}"
-* `namespace_rules` - (optional) Specifically enables or disables metric collection for specific AWS namespaces for this _AWS account only_. A list of namespaces can be found at the [available namespace rules API endpoint](https://docs.datadoghq.com/api/v1/aws-integration/#list-namespace-rules). Omitting a namespace will not change it from the default integration settings.
+* `included_namespaces` - (optional) Include only these namespaces for metrics collection for this _AWS account only_. A list of namespaces can be found at the [available namespace rules API endpoint](https://docs.datadoghq.com/api/v1/aws-integration/#list-namespace-rules). Defaults to an empty list, which means no metrics will be collected.
```

Comment:
Does this change mean that all existing consumers of this module will need to be modified during their next deployment to avoid losing metrics?